### ISSUE TITLE
Received DML 'xxx' for processing, binlog probably contains events generated with statement or mixed based replication format

### ISSUE THREAD

**Issue by shizhengchao** [ververica/flink-cdc-connectors#42]
Startup the job normally, but inserting or updating data, get the following exception:
```
2020-09-30 10:46:37.607 [debezium-engine] ERROR com.alibaba.ververica.cdc.debezium.DebeziumSourceFunction  - Reporting error:
org.apache.kafka.connect.errors.ConnectException: Received DML 'update orders set product_id = 1122 where order_number = 10001' for processing, binlog probably contains events generated with statement or mixed based replication format
at io.debezium.connector.mysql.AbstractReader.wrap(AbstractReader.java:230)
at io.debezium.connector.mysql.AbstractReader.failed(AbstractReader.java:207)
at io.debezium.connector.mysql.BinlogReader.handleEvent(BinlogReader.java:600)
at com.github.shyiko.mysql.binlog.BinaryLogClient.notifyEventListeners(BinaryLogClient.java:1130)
at com.github.shyiko.mysql.binlog.BinaryLogClient.listenForEventPackets(BinaryLogClient.java:978)
at com.github.shyiko.mysql.binlog.BinaryLogClient.connect(BinaryLogClient.java:581)
at com.github.shyiko.mysql.binlog.BinaryLogClient$7.run(BinaryLogClient.java:860)
at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.kafka.connect.errors.ConnectException: Received DML 'update orders set product_id = 1122 where order_number = 10001' for processing, binlog probably contains events generated with statement or mixed based replication format
at io.debezium.connector.mysql.BinlogReader.handleQueryEvent(BinlogReader.java:785)
at io.debezium.connector.mysql.BinlogReader.handleEvent(BinlogReader.java:583)
... 5 common frames omitted
```

**Comment by shizhengchao** [comment]
this is a bug of debezium, see https://github.com/debezium/debezium/pull/1830.

As users and programs can call SET SESSION binlog_format='STATEMENT'; SET SESSION tx_isolation='REPEATABLE-READ'; as
they wish, they are able to break debezium connector. Only MySQL 8.0.14
is fixing this, but not everyone is on that version.

so, before [debezium](https://github.com/debezium/debezium) fixes this bug,  you can upgrade  MySQL 8.0.14 to solve this problem

**Comment by leonardBang** [comment]
we used debezium 1.5.2 which has fixed this issue
